helm/alfresco-process-application/values.yaml

```diff
@@ -228,6 +228,21 @@ alfresco-process-workspace-app:
     APP_CONFIG_BPM_HOST: '{{ include "common.gateway-url" . }}'
     APP_CONFIG_APPS_DEPLOYED: '[{"name": "{{ .Release.Name }}" }]'
 
+alfresco-digital-workspace-app:
+  enabled: false
+  nameOverride: digital-workspace-app
+  ingress:
+    path: "/{{ .Release.Name }}/digital-workspace"
+  image:
+    repository: quay.io/alfresco/alfresco-digital-workspace
+    tag: develop
```

Comment:
Not sure about the tag
In Quay there's no such a thing as `7.1.0.M6`

Reply:
leave develop for now, as develop exists

when they release they will add a tag 7.1.0.M7 in quay.io as they already do for PW

and then we release the all the helm charts replacing develop with 7.1.0.M7

@mauriziovitale can you confirm this will happen for ADW as well?

Reply:
I don't think so 7.1 is strictly related to activiti and digital workspace is an app which can contain activiti. We need to agree on common release number

Reply:
but this already happens for PW   7.1.*, for example this was for M6 -> 7.1.0.M6 https://github.com/Alfresco/alfresco-process-application-deployment/blob/7.1.0.M6/helm/alfresco-process-application/values.yaml#L394 

the problem is that we are still using milestones M for M7, then we will agree on common formats. Can we enable this for ADW this release? @mauriziovitale

Reply:
alfresco-process-application is using the process 
ADW contains Content/Process/Governance/AI so I don't think is correct to use the 7.1.0.MX. @magemello @mteodori what's your opinion?

Reply:
ADW image tag in both -> application helm chart and deployment-service

Reply:
agree with Maurizio, in this case, we can't move it to 7

Reply:
@magemello do you suggest to use a different tag instead? It would be a special case though which introduces changes here and in the deployment-service(including the installation of deployment-service)

Reply:
keep the DW version

Reply:
so it's gonna be 1.5.0